### ISSUE TITLE
Moving/copy files and folders

### ISSUE THREAD

**Issue by laoneo** [joomla-projects/media-manager-improvement#442]
It should be possible to drag files and folders into other folders in MM.

**Comment by brianteeman** [comment]
absolutely. it seemed natural for this to work so i was surprised when i dragged and nothing happened
